web/test/plugins/index.js

```diff
@@ -10,6 +10,21 @@ module.exports = (on, config) => {
   });
   const db = admin.firestore();
 
+  let manipulate = (data) => {
+    const newData = {}
+    Object.entries(data).forEach((item) => {
+      switch (item[1]) {
+        case 'DATETIME[NOW]':
+          newData[item[0]] = admin.firestore.Timestamp.fromDate(new Date())
+          break;
+        default:
+          newData[item[0]] = item[1];
+      }
+    });
+    console.log(newData);
```

Comment:
We don't need console.log . I will remove it

Reply:
We need to support for nested date too

Reply:
Yes, I think this is done now 👍